### DISCUSSION TITLE
Annotating/Highlighting DjVu Files

### DISCUSSION THREAD

**Discussion by gabmifre** [sumatrapdfreader/sumatrapdf#2307] [Q&A] [answered]
Hello. Is there a way to highlight or annotate text of DjVu files? If not, is there a possibility that this can be added? Thank you.

**Comment by kjk** [comment]
No, annotations are only available in PDF. There are no plans to implement annotations in DjVu files.

**Comment by GitHubRulesOK** [comment, marked as answer]
@gabmifre 
Hmm... going against HMV there was a method in experimental versions prior to 3.2 so should still work with that version as viewer, but you will neeed to generate your own text writer.

As kjk points out this will not be revived, so for hints to work in V3.2 as viewer see https://forum.sumatrapdfreader.org/t/convert-adobe-highlight-to-smx/3384 and more at https://forum.sumatrapdfreader.org/t/annotate-highlight-add-comments-with-sumatrapdf/1092/7?u=githubrulesok but again I repeat its unsupported so your on your own if its buggy.
